tests/console/opensuse_repos.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: openSUSE-repos
+# Summary: Basic checks for openSUSE-repos.
+
+# - Install openSUSE-repos
+# - Innstall openSUSE-repos-NVIDIA on aarch64 and x86_64
+# - Import gpg keys and refresh repositories
+# - Flavors of openSUSE-repos packages correspond with flavor of openSUSE-release
+# Maintainer: Lubos Kocmman <lubos.kocman@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Utils::Architectures;
+use version_utils qw(is_tumbleweed is_leap is_leap_micro is_microos is_slowroll);
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    my $timeout = 300;
```

Comment:
Unused variable?

Reply:
Ah good point, this was originally carried over from uninstall. Will drop.